pkg/features/features.go

```diff
@@ -0,0 +1,26 @@
+package features
+
+import (
+	utilruntime "k8s.io/apimachinery/pkg/util/runtime"
+	"k8s.io/component-base/featuregate"
+)
+
+const (
+	// Add new feature gates constants (strings)
+	// Ex: SomeFeature featuregate.Feature = "SomeFeature"
+
+	BundleDeploymentHealth featuregate.Feature = "BundleDeploymentHealth"
+)
+
+var rukpakFeatureGates = map[featuregate.Feature]featuregate.FeatureSpec{
+	// Add new feature gate definitions
+	// Ex: SomeFeature: {...}
+
+	BundleDeploymentHealth: {Default: false, PreRelease: featuregate.Beta},
```

Comment:
Not sure the implications here, but I would have assumed this would be introduced as an alpha feature gate before beta (although maybe I have the meanings flipped?)

Reply:
I will change it to alpha.

Reply:
done